### ISSUE TITLE
Feature Request: Enlarge images on Media Description tab

### ISSUE THREAD

**Issue by pitbuster** [pixelfed/pixelfed#4925]
When creating alt text, it would be nice to have a way to enlarge images so we can see image details when writing the description. Right now I right click and open the image in a new tab, but that requires some knowledge and even then is quite of cumbersome.

**Comment by glitch2500** [comment]
Yes please.  I would really appreciate this feature.  It is one of the things that puts me off using Pixelfed more often.
